### DISCUSSION TITLE
Power consumption

### DISCUSSION THREAD

**Discussion by martinosk** [adonno/tagreader#92] [Q&A]
What kind of power consumption is to be expected? Would battery power be an option?

**Comment by adonno** [comment]
we are taking a battery version into consideration you can power it from a power bank but it will last 24h in its current form
We are now checking for a way to wake the ESP from sleep just before you scan the card. It will be a bit slower to send commands to HA since needs to connect to wifi first this might take some months tough.

**Reply by johnnyeightpence** [reply]
I am looking for a solution to have a RFID reader for the door with battery would be best as we dont have power supply there. Any suggestions?

**Comment by black-jmyntrn** [comment]
why wouldn't a battery like this work? 
https://www.aliexpress.us/item/3256803972189621.html
